app/models/card.rb

```diff
@@ -15,26 +19,10 @@ class Card < ActiveRecord::Base
 
   mount_uploader :image, CardImageUploader
 
-  scope :pending, -> { where('review_date <= ?', Time.now).order('RANDOM()') }
+  scope :pending, -> { where('review_date <= ?',
```

Comment:
Use the `lambda` method for multi-line lambdas.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.